Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -60,8 +60,6 @@ private void Update()
             gazeDirection = Camera.main.transform.forward;
 
             UpdateRaycast();
-
-            UpdateStabilizationPlane();
```

Comment:
I still don't get why this is being removed from GazeManager?

You can keep this in. The other script can check if the SetStabilizationPlane bool is enabled. If so, you can disable it.